packages/amplify-cli-core/src/feature-flags/featureFlags.ts

```diff
@@ -25,16 +30,16 @@ export class FeatureFlags {
 
   public static initialize = async (
     environmentProvider: CLIEnvironmentProvider,
-    useNewDefaults: boolean = false,
+    useNewDefaults = false,
     additionalFlags?: Record<string, FeatureFlagRegistration[]>,
   ): Promise<void> => {
     // If we are not running by tests, guard against multiple calls to initialize invocations
     if (typeof jest === 'undefined' && FeatureFlags.instance) {
-      throw new Error('FeatureFlags can only be initialzied once');
+      throw new Error('FeatureFlags can only be initialized once');
     }
 
     if (!environmentProvider) {
-      throw new Error(`'environmentProvider' argument is required`);
+      throw new Error('\'environmentProvider\' argument is required');
```

Comment:
I'm not a fan of this lint change. It makes the code harder to read in my opinion.

@danielleadams if you agree, could you update the linter quotation rule configuration to use `"avoidEscape": true`?

https://eslint.org/docs/rules/quotes#quotes